### PR TITLE
Create Students Controller, My Lessons Page, and Lesson Card basic styling 

### LINKED ISSUE #58: Create LessonCard Component
image

### PR DESCRIPTION
**Pull request by lilshim**
Fix #58
 image

### CODE REVIEW

File: app/assets/javascripts/constants/image_constants.es6.jsx.erb [hunk 1]
```diff
@@ -30,6 +30,16 @@
...
       }
     }
+
+    get lesson_card() {
+      return {
+        cancel: "<%= asset_path("lesson_card/cancel.png") %>",
+        location: "<%= asset_path("lesson_card/location.png") %>",
+        modify: "<%= asset_path("lesson_card/modify.png") %>",
+        person: "<%= asset_path("lesson_card/person.png") %>",
+        time: "<%= asset_path("lesson_card/time.png") %>",
```
Comment (sonia-y) [inline app/assets/javascripts/constants/image_constants.es6.jsx.erb]: These icons are probably used across the website, could you put this under a icons() method to make it more generic?

**Comment by sonia-y** [comment]
Okie, quick fix! I'll work on the styling, but don't forget to change the routes in a new pr!! :)

File: app/assets/javascripts/constants/image_constants.es6.jsx.erb [hunk 2]
```diff
@@ -30,6 +30,16 @@
...
+      }
```
Comment (sonia-y) [inline app/assets/javascripts/constants/image_constants.es6.jsx.erb]: can you add a semicolon here? I also realized I didn't put semicolons for the other ones, so can you just add it for all of them :O ty ty

**Comment by lilshim** [comment]
@sonia-y I think this branch can be closed and deleted. All of this is already merged into master from another branch

**Pull request closed without merging**